Content/src/Client/Client.fs

```diff
@@ -107,11 +107,11 @@ let query msgs =
 // The update function computes the next state of the application based on the current state and the incoming events/messages
 let update (msg : Msg) (currentModel : Model) : Model =
     match currentModel.Counter, msg with
-    | Some x, Increment ->
-        { currentModel with Counter = Some (x + 1) }
-    | Some x, Decrement ->
-        { currentModel with Counter = Some (x - 1) }
-    | _, InitialCountLoaded (Ok initialCount)->
+    | Some counter, Increment ->
```

Comment:
What if instead of `Some counter` and then using `counter.Value` we use `Some { Value = value }` and return `Some { Value = value + 1 }`?

Reply:
That'll work too, although I'm not sure how much cleaner that is. Happy to make that change though.

Reply:
I'm ok with both approaches, but see comment below